test/AutoRest.TestServer.Tests/Infrastructure/TestServerV1.cs

```diff
@@ -15,40 +15,36 @@ namespace AutoRest.TestServer.Tests.Infrastructure
 {
     public class TestServerV1 : IDisposable, ITestServer
     {
-        private static Regex _scenariosRegex = new Regex("(coverage|optionalCoverage|optCoverage)\\[(\"|')(?<name>\\w+)(\"|')\\]", RegexOptions.Compiled);
+        private static readonly Regex _scenariosRegex = new Regex("(coverage|optionalCoverage|optCoverage)\\[(\"|')(?<name>\\w+)(\"|')\\]", RegexOptions.Compiled);
 
-        private Process _process;
+        private readonly Process _process;
         public HttpClient Client { get; }
         public string Host { get; }
 
         public TestServerV1()
         {
             var portPhrase = "Server started at port ";
             var startup = Path.Combine(GetBaseDirectory(), "legacy", "startup", "www.js");
-
-            var processStartInfo = new ProcessStartInfo("node", startup);
-
-            // Use random port
-            processStartInfo.Environment["PORT"] = "0";
-            processStartInfo.RedirectStandardOutput = true;
-            processStartInfo.RedirectStandardError = true;
+            var port = "0";
```

Comment:
Undo the unrelated changes

Reply:
This matches V2 and is cleaner. We should just have 1 method for this between V1 and V2, but that's outside the scope of this PR. This code has no changes in logic, just basic housekeeping.

Reply:
Why do you think it is cleaner?

Reply:
- Uses object initializer instead of setting the values after creation.
- In V2, instead of having `port` set as `"0"` twice, it uses a variable and sets it that variable once.
- In V1, it is simply to mimic V2 for consistency between them.
- Better guarantee of the process not being null using `Debug.Assert`.
- Uses a design similar to [guard clauses](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html) to reduce unnecessary nesting.

Reply:
> Uses object initializer instead of setting the values after creation.

Which makes things inconsistent because you have to set env variable anyway.

> In V2, instead of having port set as "0" twice, it uses a variable and sets it that variable once.

In v2 environment variable is not required and in V1 local is used only once.

Reply:
But what does any of it matter? For something like this, where it isn't 'production code', nitpicking on how it *looks* doesn't make a difference at all. We both understand what the code says. I'd value spending more time implementing the rest of the features we need than being pedantic about test server code that we'll likely clean up in the future anyway. We're both working on this codebase. We can clean up anything *at any time*. As I had mentioned before Christmas break, let's focus on getting things done and being precise later. It is a much more valuable use of our time.

Reply:
That's why I say, just revert it. The change barely helps anything, is unrelated to PR at hand and makes it harder to review.

Reply:
I'll revert, but your points are invalid. You know the history of this change, thus modifying this code is not 'unrelated' to this PR in any way, since changing this code here is what initially allowed this PR to run tests. If I told you to change something and you had cosmetic changes around that thing, I'm not going to tell you to revert the cosmetic changes. That's just unnecessary work for you.